Models/New/NewCard.cs

```diff
@@ -46,7 +47,7 @@ public static class NewCard
 			bool flipPortraitForStrafe = false, bool onePerDeck = false,
 			List<CardAppearanceBehaviour.Appearance> appearanceBehaviour = null, Texture2D defaultTex = null,
 			Texture2D altTex = null, Texture titleGraphic = null, Texture2D pixelTex = null,
-			Texture2D emissionTex = null, GameObject animatedPortrait = null, List<Texture> decals = null,
+			Texture2D emissionTex = null, Texture2D altEmissionTex = null, GameObject animatedPortrait = null, List<Texture> decals = null,
```

Comment:
I would be completely fine with this PR if this weren't a breaking change that would happen like two weeks before KCM comes out and we switch to using the rewrite